.vib/mediawiki/goss/goss.yaml

```diff
@@ -1,28 +1,27 @@
 file:
-  /opt/bitnami/mediawiki/cache:
+  /bitnami/mediawiki:
     exists: true
     filetype: directory
-    mode: "0775"
-    owner: daemon
-    group: root
-  /opt/bitnami/mediawiki/images:
-    filetype: symlink
-    linked-to: /bitnami/mediawiki/images
+    mode: "2775"
+    owner: root
+  /etc/hosts:
     exists: true
-  /bitnami/mediawiki/LocalSettings.php:
-    mode: "0644"
     filetype: file
-    exists: true
-# Checks the user running the container is non-root
+    contains:
+    {{ range .Vars.hostAliases }}
+    {{ $ip := .ip }}
+    {{ range $host := .hostnames }}
+      - /{{ $ip }}.*{{$host}}/
+    {{ end }}
+    {{ end }}
+http:
+  https://mediawiki:{{ .Vars.service.ports.https }}:
```

Comment:
This test is failing, it seems due to a misconfiguration: 

```
        "test_id" : "goss-0-9",
        "name" : "HTTP https://mediawiki:444 status",
        "status" : "FAILED",
        "message" : "ERRORS: [https://mediawiki:444: status: Error: Get https://vmware-mediawiki.my/wiki/Main_Page: dial tcp: lookup vmware-mediawiki.my on 10.130.80.10:53: no such host] FAILURES: [https://mediawiki:444: status: Error: Get https://vmware-mediawiki.my/wiki/Main_Page: dial tcp: lookup vmware-mediawiki.my on 10.130.80.10:53: no such host] ",
        "duration" : 0.123,
        "location" : "goss.yaml"
```

Reply:
It seems mediawiki redirects you to the host you are using if you try accessing via IP:

```
$ curl -LIk https://mediawiki:444
HTTP/1.1 301 Moved Permanently
Date: Mon, 03 Oct 2022 10:03:29 GMT
Server: Apache
X-Content-Type-Options: nosniff
Vary: Accept-Encoding,Cookie
Expires: Thu, 01 Jan 1970 00:00:00 GMT
Cache-Control: private, must-revalidate, max-age=0
Last-Modified: Mon, 03 Oct 2022 10:03:29 GMT
Location: https://vmware-mediawiki.my/wiki/Main_Page
X-Request-Id: 73056acbb99ffbced16923d7
Content-Type: text/html; charset=UTF-8

curl: (6) Could not resolve host: vmware-mediawiki.my
```

To avoid this we would need to update the `/etc/hosts` file after knowing the LoadBalancerIP, which is more than tricky as we can't control them. 

As a workaround, try using the `no-follow-redirects: true` option in the [HTTP test](https://github.com/aelsabbahy/goss/blob/master/docs/manual.md#http). That should avoid the redirect to `vmware-mediawiki.my` and leave you with a `301` status code, which I suppose it's good enough in this case.